mozilla/fx-private-relay
# fix #1671: change From: to replies@ for premium customers

This PR fixes #1671.

How to test:
End-to-end manual test is easiest to do on the dev server:

1. Sign in as a free user
2. Generate an alias
3. Send a test email to the free alias
   * Verify the "From:" of the email is the "noreplies" address for the dev server
4. Sign in as a premium customer
5. Generate an alias
6. Send a test email to the premium alias
   * Verify the "From:" of the email is the "repies" address for the dev server
